<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-01T16:31:57 IST" name="Suite" finished-at="2022-06-01T16:31:57 IST" duration-ms="0">
    <groups>
    </groups>
  </suite> <!-- Suite -->
  <suite started-at="2022-06-01T16:31:50 IST" name="Suite (1)" finished-at="2022-06-01T16:31:57 IST" duration-ms="6839">
    <groups>
    </groups>
    <test started-at="2022-06-01T16:31:50 IST" name="Test" finished-at="2022-06-01T16:31:57 IST" duration-ms="6839">
      <class name="com.Test.TC_002_verifyRegistration">
        <test-method is-config="true" signature="BS()[pri:0, instance:com.Test.TC_002_verifyRegistration@71075444]" started-at="2022-06-01T16:31:50 IST" name="BS" finished-at="2022-06-01T16:31:50 IST" duration-ms="58" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BS -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.Test.TC_002_verifyRegistration@71075444]" started-at="2022-06-01T16:31:50 IST" name="setup" finished-at="2022-06-01T16:31:53 IST" duration-ms="2505" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_registration()[pri:0, instance:com.Test.TC_002_verifyRegistration@71075444]" started-at="2022-06-01T16:31:53 IST" name="verify_registration" finished-at="2022-06-01T16:31:53 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_registration -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.Test.TC_002_verifyRegistration@71075444]" started-at="2022-06-01T16:31:53 IST" name="tearDown" finished-at="2022-06-01T16:31:57 IST" duration-ms="4129" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verify_registration status=SUCCESS method=TC_002_verifyRegistration.verify_registration()[pri:0, instance:com.Test.TC_002_verifyRegistration@71075444] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Test.TC_002_verifyRegistration -->
    </test> <!-- Test -->
  </suite> <!-- Suite (1) -->
  <suite started-at="2022-06-01T16:31:37 IST" name="Suite (0)" finished-at="2022-06-01T16:31:50 IST" duration-ms="13531">
    <groups>
    </groups>
    <test started-at="2022-06-01T16:31:37 IST" name="Test_Chrome" finished-at="2022-06-01T16:31:44 IST" duration-ms="7302">
      <class name="com.Test.TC_001_verifyloginPage">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.Test.TC_001_verifyloginPage@5b247367]" started-at="2022-06-01T16:31:37 IST" name="setup" finished-at="2022-06-01T16:31:43 IST" duration-ms="6637" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyFBloginTest_Invaliddata()[pri:0, instance:com.Test.TC_001_verifyloginPage@5b247367]" started-at="2022-06-01T16:31:43 IST" name="verifyFBloginTest_Invaliddata" finished-at="2022-06-01T16:31:44 IST" duration-ms="467" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.Test.TC_001_verifyloginPage.verifyFBloginTest_Invaliddata(TC_001_verifyloginPage.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFBloginTest_Invaliddata -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.Test.TC_001_verifyloginPage@5b247367]" started-at="2022-06-01T16:31:44 IST" name="tearDown" finished-at="2022-06-01T16:31:44 IST" duration-ms="152" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyFBloginTest_Invaliddata status=FAILURE method=TC_001_verifyloginPage.verifyFBloginTest_Invaliddata()[pri:0, instance:com.Test.TC_001_verifyloginPage@5b247367] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Test.TC_001_verifyloginPage -->
    </test> <!-- Test_Chrome -->
    <test started-at="2022-06-01T16:31:45 IST" name="Test_Edge" finished-at="2022-06-01T16:31:50 IST" duration-ms="5681">
      <class name="com.Test.TC_001_verifyloginPage">
        <test-method is-config="true" signature="BS()[pri:0, instance:com.Test.TC_001_verifyloginPage@72bc6553]" started-at="2022-06-01T16:31:36 IST" name="BS" finished-at="2022-06-01T16:31:37 IST" duration-ms="1095" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BS -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.Test.TC_001_verifyloginPage@72bc6553]" started-at="2022-06-01T16:31:45 IST" name="setup" finished-at="2022-06-01T16:31:50 IST" duration-ms="5120" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyFBloginTest_Invaliddata()[pri:0, instance:com.Test.TC_001_verifyloginPage@72bc6553]" started-at="2022-06-01T16:31:50 IST" name="verifyFBloginTest_Invaliddata" finished-at="2022-06-01T16:31:50 IST" duration-ms="351" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.Test.TC_001_verifyloginPage.verifyFBloginTest_Invaliddata(TC_001_verifyloginPage.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFBloginTest_Invaliddata -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.Test.TC_001_verifyloginPage@72bc6553]" started-at="2022-06-01T16:31:50 IST" name="tearDown" finished-at="2022-06-01T16:31:50 IST" duration-ms="203" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyFBloginTest_Invaliddata status=FAILURE method=TC_001_verifyloginPage.verifyFBloginTest_Invaliddata()[pri:0, instance:com.Test.TC_001_verifyloginPage@72bc6553] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Test.TC_001_verifyloginPage -->
    </test> <!-- Test_Edge -->
  </suite> <!-- Suite (0) -->
</testng-results>
